app/assets/v2/js/grants/fund.js

```diff
@@ -176,7 +175,7 @@ $(document).ready(function() {
                       success: json => {
                         console.log('successfully saved subscriptionHash and signature');
                         url = json.url;
-                        $('#wait').val('false');
+                        $('#wait1').val('false');
```

Comment:
was this conflicting with some other element ?

Reply:
No, I think my branches got mixed. The browser_close PR will introduce wait1 and wait2, to ensure that a redirect will only happen once all data has been processed and saved. The fund.html file from the branch I think was introduced to this branch, I was saw wait1 and wait 2 so just went with it because I know that is where the code is going. I didnt think much about it, honestly. I will revert for this PR.